setup.py

```diff
@@ -24,86 +17,25 @@
     'Operating System :: Microsoft :: Windows',
     'Operating System :: POSIX',
     'Operating System :: Unix',
-    'Programming Language :: C',
-    'Programming Language :: Python :: 2',
-    'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.3',
     'Topic :: Scientific/Engineering',
     'Topic :: Software Development',
-    'Topic :: Text Processing :: Linguistic',]
+    'Topic :: Text Processing :: Linguistic',
+]
 
-README_contents = open(os.path.join(current_dir, 'README.txt')).read()
+current_dir = os.path.dirname(__file__)
+README_contents = open(os.path.join(current_dir, 'README.md')).read()
 doclines = README_contents.split("\n")
+dependencies = ['ftfy >= 4', 'msgpack-python', 'langcodes']
+if sys.version_info < (3, 4):
+    dependencies.append('pathlib')
 
 
-class SimpleCommand(Command):
-    """
-    Get the boilerplate out of the way for commands that take no options.
-    """
-    user_options = []
-
-    def initialize_options(self):
-        pass
-
-    def finalize_options(self):
-        pass
-
-
-class BuildDatabaseCommand(SimpleCommand):
-    description = "Build the word frequency database from raw data"
-    def run(self):
-        from wordfreq.build import load_all_data
-        load_all_data()
-
-
-class DownloadDatabaseCommand(SimpleCommand):
-    description = "Download the built word frequency database"
-    user_options = []
-
-    def run(self):
-        transfer.download_db()
-
-
-class DownloadRawDataCommand(SimpleCommand):
-    description = "Download the raw wordlist data"
-    user_options = []
-
-    def run(self):
-        transfer.download_and_extract_raw_data()
-
-
-class UploadDataCommand(SimpleCommand):
-    description = "Upload the raw data and database"
-    user_options = []
-
-    def run(self):
-        transfer.upload_data()
-
-
-class CustomInstallCommand(install):
-    def run(self):
-        install.run(self)
-        if not os.path.exists(config.DB_FILENAME):
-            self.run_command('download_db')
-
-
-class CustomDevelopCommand(develop):
-    def run(self):
-        develop.run(self)
-        if not os.path.exists(config.DB_FILENAME):
-            self.run_command('download_db')
-
-
-requirements = ['ftfy >= 4']
-if sys.version_info.major == 2:
-    requirements.append('functools32')
-
 setup(
     name="wordfreq",
-    version=config.VERSION,
+    version='1.0b2',
     maintainer='Luminoso Technologies, Inc.',
-    maintainer_email='dev@luminoso.com',
+    maintainer_email='info@luminoso.com',
```

Comment:
I got "info@luminoso.com" from another setup.py. What should it be?

I don't think we want it to be dev@, because I don't think we want random people who use this package e-mailing that list.